src/unity/lib/annotation/annotation_base.cpp

```diff
@@ -24,9 +31,10 @@ AnnotationBase::AnnotationBase(const std::shared_ptr<unity_sframe> &data,
 
 void AnnotationBase::annotate(const std::string &path_to_client) {
   visualization::process_wrapper aw(path_to_client);
-  /* TODO: Easier to Implement when working on the Front End. Hold off
-   * implementation until front end PR. Will be more clear what to implement
-   * there */
+
+  aw << this->__serialize_proto<annotate_spec::MetaData>(this->metaData())
+            .c_str();
+
   while (aw.good()) {
     break;
```

Comment:
Don't we want to keep the connection open now? Break will exit the loop and allow the annotate call to return.